data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -114,66 +106,14 @@ fn add_path(url: &Uri, path: &str) -> Uri {
         }
         None => PathAndQuery::from_str(path),
     }
+    // TODO: APMSP-18190
```

Comment:
nit: For the `TODO` comments we should also include a sentence describing the todo for those who don't have access to Jira. Something like "Properly handle non-OK states to prevent possible panics" or something like that.